Python/Tests/DebuggerUITests/DebugProjectUITests.cs

```diff
@@ -123,14 +111,11 @@ public class DebugProjectUITests {
         public void DebugPythonCustomInterpreter(VisualStudioApp app, bool useVsCodeDebugger, DotNotWaitOnNormalExit optionSetter) {
             var pyService = app.ServiceProvider.GetUIThread().Invoke(() => app.ServiceProvider.GetPythonToolsService());
             using (new PythonExperimentalGeneralOptionsSetter(pyService, useVsCodeDebugger: useVsCodeDebugger)) {
-                if (useVsCodeDebugger) {
-                    KillPythonProcess(TimeSpan.FromSeconds(30));
-                }
-
                 var sln = app.CopyProjectForTest(@"TestData\RelativeInterpreterPath.sln");
                 var project = app.OpenProject(sln, "Program.py");
                 var interpreterPath = Path.Combine(PathUtils.GetParent(sln), "Interpreter.exe");
-                File.Copy(PythonPaths.Python27.InterpreterPath, interpreterPath, true);
+                var interpreter = PythonPaths.Python27 ?? PythonPaths.Python27_x64;
```

Comment:
Add an `interpreter.AssertInstalled()` call before using it, and put it first thing in the test.